src/curobo/wrap/reacher/motion_gen.py

```diff
@@ -1946,8 +1954,8 @@ def _plan_js_from_solve_state(
                     og_solve_time = traj_result.solve_time
 
                     scaled_dt = torch.clamp(
-                        torch.max(traj_result.optimized_dt[traj_result.success]),
-                        self.trajopt_solver.interpolation_dt,
+                        torch.max(traj_result.optimized_dt[traj_result.success]) * self.finetune_dt_scale,
```

Comment:
This is I think a minor fix as finetune_dt_scale must have been applied

Reply:
Currently, we only use finetune_dt_scale only for pose planning. Joint space planning was implemented to facilitate going to a home configuration after doing manipulation tasks.

The finetune attempts could be implemented in plan_single_js as well. Thanks for bring this up.

Reply:
Should I remove this from the PR?

Reply:
Yes, can you remove this?